MovieDB/MovieDB/Scenes/Dashboard/Cells/Category Item/CategoryItemCollectionViewCell.swift

```diff
@@ -0,0 +1,45 @@
+import UIKit
+import Reusable
+
+fileprivate enum ConstantCategoryItemCollectionCell {
+    static let fontSizeCategoryTitle = CGFloat(12)
```

Comment:
Don't assign a converting Int to CGFloat when declared a variable. Should declare the value type of it.
static let categoryTitleFontSize: CGFloat = 12